spec.md

```diff
@@ -281,6 +281,27 @@ help intermediate gateways determine how to route the events.
 As defined by the term [Data](#data), CloudEvents MAY include domain-specific
 information about the occurrence. When present, this information will be
 encapsulated within the `data` attribute.
+The `dataref` attribute can be used to reference another location where this
+information is stored. Known as the "Claim Check Pattern", the `dataref`
+attribute can be used for different purposes:
```

Comment:
- s/can/MAY/  I think
- s/different purposes/a variety of purposes, including:/